pkg/server/sessions/session.go

```diff
@@ -53,12 +55,18 @@ type Session struct {
 	creationTime             time.Time
 	lastActivityTime         time.Time
 	transactions             map[string]transactions.Transaction
-	paginatedDocumentReaders map[string]*PaginatedDocumentReader // map from searchID to sql.RowReader objects
+	paginatedDocumentReaders *cache.LRUCache // track searchID to sql.RowReader objects
```

Comment:
`documentReaders *cache.LRUCache // track searchID to document.DocumentReader objects` ?

here is doesn't need to know it's a paginated reader